R/pkg/R/context.R

```diff
@@ -295,6 +295,23 @@ setCheckpointDirSC <- function(sc, dirName) {
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(dirName))))
 }
 
+#' Set a human readable description of the current job.
+#'
+#' Set a description that is shown as a job description in UI.
+#'
+#' @rdname setJobDescription
+#' @param value The job description of the current job.
+#' @export
+#' @examples
+#'\dontrun{
+#' setJobDescription("This is a example job.")
+#'}
+#' @note setJobDescription since 2.3.0
```

Comment:
Just for sure, 

![2017-06-22 11 04 44](https://user-images.githubusercontent.com/6477701/27414185-acad6f34-573a-11e7-9f63-2c1ee4562ac0.png)